CHANGELOG.md

```diff
@@ -43,6 +43,26 @@ with gr.Blocks() as demo:
     gr.Video(("video.mp4", "captions.srt"))
 ```
 
+### HighlightedImage Component
```

Comment:
Suggestion that I also put in Slack: how about `AnnotatedImage` component?

Reply:
Done